<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDcAcOpencvClass</class>
 <widget class="QMainWindow" name="QDcAcOpencvClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDcAcOpencv</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>621</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>511</width>
        <height>281</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButtonStart">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>351</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxId">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>161</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>id</string>
      </property>
      <widget class="QLineEdit" name="lineEditKey">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditValue">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>key</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>value</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonUpdate">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>200</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>add/update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDelete">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>270</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qdcacopencv.qrc"/>
 </resources>
 <connections/>
</ui>
